pkg/flowkit/services/blocks.go

```diff
@@ -50,6 +50,15 @@ func NewBlocks(
 	}
 }
 
+type IBlocks interface {
```

Comment:
Uff you from Java? :D 
I think it's a good idea to make an interface for services, I think it would make sense to even have a single interface in front of all services, I don't like how I designed this a while ago. It would however be a big breaking change so I think it should be as a separate PR that then allows this tests PR.

Reply:
> Uff you from Java? :D

Oh no is that a Javaism I'll come up with a better name 🤣
I'm having troubles thinking of a way to refactor the `Services` struct to be be defined by a single interface. Here is what I could think of https://github.com/onflow/flow-cli/compare/neil/refactor-services-for-testing

Reply:
I think I would rather refactor that the whole service layer has one interface and then methods like `GetTransaction`, `GetBlock` etc. I think we should first discuss this in an issue, so open an issue, write your proposal, I will leave comments and lets define things first there and then move on with implementation. The issue can be generic "refactor of services to use an interface" not just for testing, since this will improve other things as well.